protocols/src/N2NP.nroff

```diff
@@ -718,14 +717,14 @@ each other: therefore, no routing is needed.
 
 But, if those two nodes are not directly connected to each other, node A must 
 determine the best route to reach node Z. In order to do that, and with all the 
-information about all the nodes on the network (See PEERS), node A must run an 
-heuristic algorithm on all his peers node (physically connected) with the score 
+information about all the nodes on the network (See PEERS), node A must run a 
+heuristic algorithm on all his peer nodes (physically connected) with the score 
 of each node as weight.
 
 In order not to loose time in the process, the depth used for the
-algorithm should be minimal: node A do not need to determine an exact
-perfect path all the way to node Z, but rather choose between his direct
-peers, the one that will be more suited to route this packet.
+algorithm should be minimal: node A does not need to determine an exact
+perfect path all the way to node Z, but rather choose the peer that will be
+most suited to route this packet.
```

Comment:
`[...] between it's direct peers, [...]` is needed, because a node cannot directly send a packet to a non-neighboring node.

Reply:
I agree that a `direct` would make it more clear, but I would rather build that into my version as it make the sentence as whole easier to understand, this would also mean that the comma before `between` cannot be done, as `between` is not part of the sentence anymore as my version completely removes this extra complement

Reply:
But why removing it if it's clearer?

Reply:
I think you misunderstood, I am in favor of adding the `direct` back in, just into the rewritten version from my commit rather than going back to the previous version